internal/node/monitor/client.go

```diff
@@ -280,42 +279,34 @@ func (c *activitypubClient) TargetState(_ *config.Parameters) (uint64, uint64) {
 }
 
 // LatestState returns the latest state of the Kafka consuming process
-func (c *activitypubClient) LatestState(ctx context.Context) (uint64, uint64, error) {
-	consumer := c.activitypubClient.GetKafkaConsumer()
-	// Create a very short timeout for the poll operation
-	pollCtx, cancel := context.WithTimeout(ctx, 1000*time.Millisecond)
-	defer cancel()
-
-	// Use PollFetches with the short timeout
-	fetches := consumer.PollFetches(pollCtx)
-
-	// Check if the poll operation timed out
-	if errors.Is(pollCtx.Err(), context.DeadlineExceeded) {
-		return 0, 0, fmt.Errorf("poll operation timed out, possible consumer issue")
-	}
-
-	if errs := fetches.Errors(); len(errs) > 0 {
-		for _, e := range errs {
-			fmt.Printf("consumer poll fetch error: %v\n", e.Err)
-		}
-
-		return 0, 0, fmt.Errorf("consumer poll fetch error: %v", fetches.Errors())
-	}
+func (c *activitypubClient) LatestState(_ context.Context) (uint64, uint64, error) {
+	// consumer := c.activitypubClient.GetKafkaConsumer()
+	//// Create a very short timeout for the poll operation
+	// pollCtx, cancel := context.WithTimeout(ctx, 1000*time.Millisecond)
+	// defer cancel()
+	//
+	//// Use PollFetches with the short timeout
+	// fetches := consumer.PollFetches(pollCtx)
+	//
+	//// Check if the poll operation timed out
+	// if errors.Is(pollCtx.Err(), context.DeadlineExceeded) {
+	//	return 0, 0, fmt.Errorf("poll operation timed out, possible consumer issue")
+	// }
+	//
+	// if errs := fetches.Errors(); len(errs) > 0 {
+	//	for _, e := range errs {
+	//		fmt.Printf("consumer poll fetch error: %v\n", e.Err)
+	//	}
+	//
+	//	return 0, 0, fmt.Errorf("consumer poll fetch error: %v", fetches.Errors())
+	// }
```

Comment:
The health check logic here must not be missing

Reply:
Will Address this using a new PR based on this one.